### DISCUSSION TITLE
Is it safe to install Open Shell in Windows 11?

### DISCUSSION THREAD

**Discussion by Dave-Rado** [Open-Shell/Open-Shell-Menu#1594] [Q&A] [answered]
Hi

According to [this article](https://www.makeuseof.com/open-shell-in-windows-11/) it is, but on the [downloads page](https://github.com/Open-Shell/Open-Shell-Menu) , Windows 11 isn't listed as a supported OS, which is odd if it's really safe to use with Win 11. Any ideas?

Dave

**Comment by ge0rdi** [comment, marked as answer]
You should use [latest](https://github.com/Open-Shell/Open-Shell-Menu/releases) Open-Shell build on Windows 11.
Official release (4.4.170) doesn't support Windows 11 yet.

**Reply by Dave-Rado** [reply]
Many thanks, @ge0rdi. Just to clarify, is release 4.4.189 in beta and therefore potentially buggy? If not, I was wondering why it still has a status of "pre-release".

Just want to check whether installing it on my mum's computer is a good idea or whether  it might be risky.

Thanks again.

Dave

**Reply by ge0rdi** [reply]
We don't have reported any issues with that build.
In general we are trying to fix any regressions in pre-release builds, so I believe they are safe to use.
There are few minor things I'd like to polish before releasing it as official (though no eta for that).

If you want to use `Open-Shell` on Windows 11 then you simply have to use pre-release builds (or live with improper support in official release).

**Reply by Dave-Rado** [reply]
Thanks, @ge0rdi. I've just installed it but I can't see any way of ungrouping the WiFi, Sound and Battery Charging icons in the notification area of the Taskbar. That was my biggest reason for wanting to install Open-shell, because having those three buttons grouped so that clicking on one of them brings up a dialog for all three at once is horrible. I had assumed that Open-shell would allow me to configure the Taskbar notification area to behave as it did in earlier versions of Windows. Is this possible?

Dave

**Reply by ge0rdi** [reply]
Nope. That's not what `Open-Shell` does. I'm sorry.

**Reply by Dave-Rado** [reply]
What a shame - it does allow one to configure the start menu and File Explorer to look and behave like earlier versions of Windows - and now that the Windows 11 Systray has become so horrible I'd assumed it would allow one to fix that too. Oh well.
